### PR TITLE
Updated Slack icon at 'footer' and 'right side' of Blogs page

### LINKED ISSUE #42847: Slack icon at 'footer' and 'right side' of Blogs page is outdated, and needs to be updated
**This is a Feature/Enhancement Request**

**What would you like to be enhanced**
The Slack icon at footer and right side of the [Blogs Page](https://kubernetes.io/blog/) is outdated and needs to be replaced with current slack icon like the one present in [Home page](https://kubernetes.io/) of K8s website

Screenshot (795)

**Additional information**
This issue is present only in Blogs page, rest of the pages have updated slack icon

### PR DESCRIPTION
**Pull request by Manish-hr14**
fix #42847

### CODE REVIEW

**Review by Gauravpadam — COMMENTED** [review]
lgtm, thanks for the PR @Manish-hr14!

It's okay to totally remove the script too, but this works as well

**Comment by Manish-hr14** [comment]
> lgtm, thanks for the PR @Manish-hr14!
> 
> It's okay to totally remove the script too, but this works as well

but in netlify preview window still showing old logo. but in my local system working fine

**Comment by Gauravpadam** [comment]
on second thought, we should remove the script to avoid future complaints

**Comment by Manish-hr14** [comment]
> on second thought, we should remove the script to avoid future complaints

oh noted let waits for another Reviewers.

File: layouts/blog/baseof.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     {{ end }}
     {{ partial "head.html" . }}
     <title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
-    <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+    <script defer src="https://use.fontawesome.com/releases/v6.5.1/js/all.js"></script>
```
Comment (sftim) [inline layouts/blog/baseof.html:9]: Could we just remove this line? If that works, I think it's a better change.

**Review by Manish-hr14 — COMMENTED** [review]
i have removed the line.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

**Pull request merged**